shared/inspector/index.tsx

```diff
@@ -137,19 +137,21 @@ const RowList = observer(function RowList({
 
   if (disableVirtualisedRendering) {
     const rows = maxHeight ? items.slice(0, maxHeight + 1) : items;
-    const showExpandedButton =
+    const showExpandBtn =
       maxHeight &&
       (rows.length > maxHeight ||
         rows.reduce((s, r) => s + (r.height ?? 1), 0) > maxHeight);
 
+    setShowExpandBtn(!!showExpandBtn);
```

Comment:
You can't call a state update directly inside the render function like this. Why does the 'show more' button need to move?